packages/react-composites/src/composites/CallComposite/CallControls.tsx

```diff
@@ -12,6 +19,11 @@ import { usePropsFor } from './hooks/usePropsFor';
 import { devicePermissionSelector } from 'calling-component-bindings';
 import { useAdaptedSelector } from './hooks/useAdaptedSelector';
 
+const localVideoViewOption = {
```

Comment:
Should these be the default options that can then be overridden by Contoso?

Seeing how we want these same values for all of our samples, composite, stories etc, it feels like a good candidate for a default.

Reply:
Yeah I considered that, looking overall of how we're exposing the UX customization for composite now, I think we do not have a pattern yet, if we expose this property from this level, and the parent component does not expose further out, then it's not good. so this PR will just add these two functionality to composite and sample. We should in the future have a overall scan of what UX functionality to be exposed from composite :)